files/Aboot/boot0.j2

```diff
@@ -120,6 +120,17 @@ platform_specific() {
     fi
 }
 
+# check the hash file in the image, and determine to install or just skip
+GIT_REVISION=$(unzip -p "$swipath" .imagehash)
+LOCAL_IMAGEHASH=$(cat $image_path/.imagehash 2>/dev/null || true)
```

Comment:
> 2>/dev/null || true [](start = 44, length = 20)

No need to handle stderr and return code.
[https://www.gnu.org/software/bash/manual/bashref.html#Command-Substitution](https://www.gnu.org/software/bash/manual/bashref.html#Command-Substitution)